tests/lib/SimpleSAML/Utils/AttributesTest.php

```diff
@@ -95,10 +92,8 @@ public function testGetExpectedAttributeEmptyAttribute()
             'attribute' => [],
         ];
         $expected = 'attribute';
-        $this->expectException(
-            \SimpleSAML\Error\Exception::class,
-            "Empty attribute '".$expected."'.'"
-        );
+        $this->expectException(\SimpleSAML\Error\Exception::class);
+        $this->expectExceptionMessage("Empty attribute '".$expected."'.'");
         Attributes::getExpectedAttribute($attributes, $expected);
```

Comment:
Not sure in what PHPunit-version this was allowed, but today it's split in two calls (actually for years already)